src/server.ts

```diff
@@ -37,17 +37,17 @@ export async function serve(options: ServeOptions, createLangHandler: () => Lang
 	rewriteConsole();
 
 	if (cluster.isMaster) {
-		console.error(`Master node process spawning ${options.clusterSize} workers`)
-		for (let i = 0; i < options.clusterSize; ++i) {
-			const worker = cluster.fork().on('disconnect', () => {
-				console.error(`worker ${worker.process.pid} disconnect`)
-			});
-		}
-
+		console.error(`Master node process spawning ${options.clusterSize} workers`);
+		cluster.on('online', worker => {
+			console.error(`Worker ${worker.id} online`);
+		});
 		cluster.on('exit', (worker, code, signal) => {
-			const reason = code === null ? signal : code;
-			console.error(`worker ${worker.process.pid} exit (${reason})`);
+			console.error(`Worker ${worker.id} exited from signal ${signal} with code ${code}, restarting`);
+			cluster.fork();
```

Comment:
Would this cause issues with legitimate killing of workers? E.g., in dev you kill the master and the workers respawn indefinitely.